### PR TITLE
user.dir correction for CYGWIN

### PR DESCRIPTION
**Pull request by tenochdlc**

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @tenochdlc,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by marcospereira** [comment]
@tenochdlc could you please squash the two commits?

**Comment by tenochdlc** [comment]
@marcospereira    Fixed.  Ta

File: framework/src/sbt-plugin/src/main/scala/play/sbt/PlaySettings.scala [hunk 1]
```diff
@@ -235,7 +235,7 @@ object PlaySettings {
     },
 
     // Adds the Play application directory to the command line args passed to Play
-    bashScriptExtraDefines += "addJava \"-Duser.dir=$(cd \"${app_home}/..\"; pwd -P)\"\n",
+    bashScriptExtraDefines += "addJava \"-Duser.dir=$(realpath \"$(cd \"${app_home}/..\"; pwd -P)\"  $(is_cygwin && echo \"fix\"))\"\n",
```
Comment (benmccann) [inline framework/src/sbt-plugin/src/main/scala/play/sbt/PlaySettings.scala]: echo "fix" looks like debug output that maybe you don't mean to commit?

Comment (tenochdlc) [inline framework/src/sbt-plugin/src/main/scala/play/sbt/PlaySettings.scala]: On cygwin the code: "$(is_cygwin && echo \"fix\")"  returns the string "fix", which is used as the second parameter for "realpath". This second parameter is requiered on cygwin to translate the path to Windows format. When not on Cygwin, that code will not return anything, so no Windows path translation will be done by "realpath".

Please have a look at:
sbt-native-packager/src/main/resources/com/typesafe/sbt/packager/archetypes/bash-template

Comment (tenochdlc) [inline framework/src/sbt-plugin/src/main/scala/play/sbt/PlaySettings.scala]: https://github.com/sbt/sbt-native-packager

**Pull request merged**